modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheLifecycleAwareSelfTest.java

```diff
@@ -234,6 +222,13 @@ private TestInterceptor() {
             super(CACHE_NAME);
         }
 
+        /** {@inheritDoc} */
+        @Override public void start() {
```

Comment:
Same goes here - we don't need this method - we just need to add `Serializable` marker to `TestLifecycleAware`.

Reply:
`public interface CacheInterceptor<K, V> extends Serializable {`

Reply:
`expCacheName` is not serialized by default, because supertype is not serializable.

from javadoc (Serializable)
`To allow subtypes of non-serializable classes to be serialized, the subtype may assume responsibility for saving and restoring the state of the supertype's public, protected, and (if accessible) package fields. The subtype may assume this responsibility only if the class it extends has an accessible no-arg constructor to initialize the class's state.`

Reply:
You are correct! Fix changed according to your comments. Good catch. Thanks!